types/draft-js/draft-js-tests.tsx

```diff
@@ -98,7 +99,9 @@ class RichEditorExample extends React.Component<{}, { editorState: EditorState }
         return getDefaultKeyBinding(e);
     }
 
-    handleKeyCommand = (command: string, editorState: EditorState) => {
+    handleKeyCommand = (command: string, editorState: EditorState, eventTimeStamp: number) => {
+        console.log(command, eventTimeStamp)
```

Comment:
Please remove this `console.log` statement

Reply:
done